### PR TITLE
Angular: Add global CSF3 renderer

### PR DESCRIPTION
**Pull request by yngvebn**
Issue: https://github.com/storybookjs/storybook/issues/15498

## What I did

Added a basic global renderer for Angular and CSF3
Enabled previewCsfV3 for the angular-cli example storybook

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

Added a couple of stories to examples/angular-cli

### CODE REVIEW

**Comment by yngvebn** [comment]
@shilman / @ThibaudAV I'm struggling with getting the tests to run green locally, even though I haven't touched any of the files or tests it's complaining about... Would greatly appreciate if someone would help out.. The same goes for this PR: https://github.com/storybookjs/storybook/pull/15743

File: examples/angular-cli/src/stories/csf3/input-with-play.stories.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { InputComponent } from './sb-input.component';
+
+export default {
+  title: 'CSF3/Input',
```
Comment (ThibaudAV) [inline examples/angular-cli/src/stories/csf3/input-with-play.stories.ts:6]: maybe `Core / CSF3 / Input` ?

Comment (yngvebn) [reply examples/angular-cli/src/stories/csf3/input-with-play.stories.ts:6]: Would "Preview" be a fitting prefix perhaps?

Comment (ThibaudAV) [reply examples/angular-cli/src/stories/csf3/input-with-play.stories.ts]: - In core I tried to place the stories showing and testing the native functionality of sotrybook
- basics it's more for examples of storybook use 
- addons for addons
..
so if CSF3 is to become something native in storybook main set, I can see it in `core/...` 

but it's up to you, knowing that ;)

**Review by shilman — APPROVED** [review]
Awesome! Thanks @yngvebn 🚀

**Comment by yngvebn** [comment]
> Awesome! Thanks @yngvebn 🚀 

You bet!

**Pull request merged**